packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1933,3 +1933,15 @@ def migrate_usercode_v5_to_v4() -> list[Callable]:
         drop("origin_node_side_type"),
```

Comment:
ya I can. I didn't see any migrate decorators in the codebase for any client-side code so that's why I wasn't sure, but it looks like we call migrations on all args and kwargs so I'll add.